core/app/[locale]/(default)/account/(tabs)/addresses/_components/customer-edit-address.tsx

```diff
@@ -4,6 +4,7 @@ import { FormFieldsFragment } from '~/client/fragments/form-fields';
 import { graphql, ResultOf } from '~/client/graphql';
 import { ExistingResultType } from '~/client/util';
 
+import { bypassReCaptcha } from '../../../../../../../lib/bypass-recaptcha';
```

Comment:
Is there a reason we don't use `~/` here?

Reply:
That was an auto import, let me update.